src/pages/chapter/index.tsx

```diff
@@ -40,11 +51,16 @@ function RecipientsCardsGrid() {
   }
 
   return (
-    <SimpleGrid columns={{ base: 1, md: 2, lg: 5 }} my="5" spacing="5">
+    <Stack spacing="5" h="90vh" maxH="600px" overflowY="scroll" pr="2" py="5">
```

Comment:
Nit: Remove `h="90vh`, set `overflowY` to `auto` to only show scrollbar when content overflows

Reply:
Do we still need to specify the max height for the list of recipients? If we specify just the `overflowY` with not max height, it will simply list all the recipients on the page without any overflow.

Reply:
yeah keep the maxHeight